dodola/core.py

```diff
@@ -74,6 +74,8 @@ def adjust_quantiledeltamapping_year(
     halfyearwindow_n : int, optional
         Half-length of the annual rolling window to extract along either
         side of `year`.
+    include_quantiles : bool, optional
+        Whether or not to output quantiles in bias corrected output.
```

Comment:
Maybe mention what coordinate/variable will be added if `include_quantiles=True`. I could see someone reading this as getting an extra returned array, or attribute, or maybe a change in the actual calculation. I think would be very helpful to be clear about where this new data is.